### ISSUE TITLE
Quarto does not render with some {gt} table IDs

### ISSUE THREAD

**Issue by AlbertRapp** [quarto-dev/quarto-cli#3247]
### Bug description

I'm not sure whether that's a {gt} or a Quarto problem. Since the {gt} code itself runs, I put it here.

Using IDs in {gt} tables causes issues with rendering a Quarto document. I don't know what the error means so I'll just put it here.

```
Error running filter /opt/quarto/share/filters/quarto-pre/quarto-pre.lua:
/opt/quarto/share/filters/quarto-pre/quarto-pre.lua:2555: attempt to concatenate a nil value (local 'v')
stack traceback:
	/opt/quarto/share/filters/quarto-pre/quarto-pre.lua:2562: in function </opt/quarto/share/filters/quarto-pre/quarto-pre.lua:2560>
```

The code that caused this error was (that's also the whole file)

```
ggplot2::mpg |> 
  gt::gt(id = 'myTbl')
```

## Version info

RStudio: 2022.07.2 Build 576
OS: Ubuntu 20.04.5 LTS
Quarto:  1.2.267

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by cscheid** [comment]
I can reproduce with

````
---
title: "3247"
---

```{r}
library(ggplot2)
library(gt)

ggplot2::mpg |> 
  gt::gt(id = 'myTbl')
```
````

**Comment by cscheid** [comment]
This is the same bug as #2370. We've fixed it in `v1.3`, but we're being very conservative about `v1.2` changes right now to minimize regressions ahead of an upcoming release. The fix should be available in installers in about a week.

**Comment by AlbertRapp** [comment]
Awesome. I look forward to v1.3 now.
